src/app/components/CopyToClipboard/index.tsx

```diff
@@ -44,12 +84,8 @@ export const CopyToClipboard: FC<CopyToClipboardProps> = ({ label, value }) => {
       onClick={handleCopyToClipboard}
       sx={{ display: 'inline-flex', alignItems: 'center' }}
     >
-      {label || value}
-      <Tooltip arrow onOpen={hideTooltip} open={isCopied} placement="top" title={t('clipboard.success')}>
-        <IconButton color="inherit" aria-label={ariaLabel}>
-          <ContentCopyIcon fontSize="small" sx={{ color: COLORS.brandDark }} />
-        </IconButton>
-      </Tooltip>
+      {label || (isMobile ? trimLongString(value) : value)}
```

Comment:
OK then, how about having a wrapper component, `AbbreviatingCopyToClipboard`, for adding that  funcionality?

Reply:
Or making it an opt-in optional feature via a prop?

Reply:
That's all odd. I'd abbreviate in AccountLink and similar components

Reply:
Yeah, we are already abbreviating it in AccountLink. The problem is that AccountLink doesn't provide the copy-to-clipboard functionality, because the current implementation of CopyToClipboard can't handle links. (Since it expects a string label or value, and not a component.)  So we are having the link first, and the separately the button. I find it inconvenient and error-prone having to specify the value twice, so it would be nice to have a common wrapper component, one way or the other.

Reply:
I think we will have quite a bit of variation in displaying the information e.g.
- oasis1 aaaa ... dddd 📋 (consensus addr)
- 0xaaaa...dddddd 📋 (evm addr)
- aaaa...dddd 📋 (consensus tx hash)
- ValidatorNam... 📋 (consensus validator addr)

so in my mind copied value isn't closely tied to displayed value (and will be error-prone)